### PR TITLE
Mix integration

### PR DESCRIPTION
**Pull request by msoedov**
Personally I found current readme instructions a bit flustrating. I want to share with you my `mix` integration changes which suppose to be more friendly for language newcomers.

### CODE REVIEW

File: test/test_helper.exs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env elixir
+ExUnit.start
```
Comment (apieum) [inline test/test_helper.exs]: Does this is ok with about_testing ? (see line 4 Koans.About_testing.start_exUnit! )

**Comment by apieum** [comment]
I think it's a good idea, it will resolve elixir version problems we had before...

File: test/test_helper.exs (hunk 1 above)
Comment (msoedov) [inline test/test_helper.exs]: I'll double check that

Comment (msoedov) [inline test/test_helper.exs]: I've checked that - about_testing works the same as before

```shell
➜  elixir-koans git:(mix) mix test
** (Koans.MeditateWarning) Please meditate: your module should use ExUnit.Case instead FakeUnit.Case
    utils/Koans.ex:30: Koans.meditate/1
    about_testing.exs:8: (module)


```

**Comment by msoedov** [comment]
@apieum 

Merge conflict is resolved now

**Comment by apieum** [comment]
Great, thanks !

**Pull request merged**